### ISSUE TITLE
BUG: Opening table defaults sorts on first column

### ISSUE THREAD

**Issue by jancoow** [beekeeper-studio/beekeeper-studio#1417]
When viewing data of a table, it default sorts on the first column. This will put a huge load when opening a large table, and in my case the query will easily take 10 minutes +. 

It should not automatically select a column to sort. Also, when closing the tab the query won't be canceled so the load will continue on the database. There is no way to see running queries in the GUI. 

Using postgresql.

**Comment by rathboma** [comment]
I agree with this. You are totally right. We should be able to fix this fairly easily, although some db engines don't support limit/offset without a sort order, so will require some workarounds

**Comment by michi-zuri** [comment]
Mabye using limit and offset is not the best pagination solution: https://use-the-index-luke.com/no-offset
Which db engines are you referring to @rathboma ?
